packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -14,7 +17,16 @@ jest.mock('@salesforce/core', () => {
 describe('test project paths', () => {
   const hasRootWorkspaceStub = jest.spyOn(workspaceUtils, 'hasRootWorkspace');
   const FAKE_WORKSPACE = '/here/is/a/fake/path/to/';
+  const FAKE_WORKSPACE_INSTANCE: any = {};
   const FAKE_STATE_FOLDER = '.sfdx';
+  const FAKE_METADATA = 'metadata';
+  const FAKE_ORGS = 'orgs';
+  const FAKE_TEST_RESULTS = 'testresults';
+  const FAKE_APEX = 'apex';
+  const FAKE_APEX_DB = 'apex.db';
+  const FAKE_DEBUG = 'debug';
+  const FAKE_LOGS = 'logs';
+  const FAKE_LWC = 'lwc';
```

Comment:
Rather then duplicate these here we should export the const values from paths.ts

Reply:
yess. I was going to ask you about it - to do the same we're doing with tools. Thank you!